.github/workflows/pull-request.yml

```diff
@@ -334,6 +334,14 @@ jobs:
 
       - name: Annotate ESLint results for modified files
         run: yarn run eslint -c ./.eslintrc.changed.js --ext .js --ext .jsx --no-error-on-unmatched-pattern --format ./script/github-actions/eslint-annotation-format.js ${{ steps.get-lint-files.outputs.JSFILES }}
+      
+      - name: Check for broken imports
+        id: check-for-broken-imports
+        continue-on-error: false
+        uses: ./.github/workflows/check-imports
+        with:
+          delimiter: ','
```

Comment:
Looks like these "with" args may be leftover from a prior copied step - not seeing them used in the code. Can we remove these?